modules/administration-guide/examples/snip_che-admonition-product-json-link.adoc

```diff
@@ -0,0 +1 @@
+NOTE: The values defined in the `/branding/product.json` file override the link:https://github.com/che-incubator/che-code/blob/main/code/product.json[default values].
```

Comment:
The reference is actual for the upstream docs.
The corresponding reference for the downstream docs is: https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/code/product.json

Reply:
Checked. This is already in the downstream docs repo.